src/nt_users_tool/yamlprocessing.py

```
@@ -0,0 +1,31 @@
+import yaml
+
+
+def output_file_path(config_file_name: str):
```

Currently, you open the file and read from it twice, which is not optimal. Here is what I suggest:
* have 1 function to read from the file and just return the content
* have 1 function to get a particular config from the content. Inside this function, we should handle the possibility that the config that we are currently trying to get is missing! In this case, let's throw an exception and exit the script right away.

What do you think?

---

The current functions will be removed. Your solution is adopted.